rocky/reports/views/report_overview.py

```diff
@@ -30,7 +30,7 @@ class ReportHistoryView(BreadcrumbsReportOverviewView, OctopoesView, ListView):
     Shows all the reports that have ever been generated for the organization.
     """
 
-    paginate_by = 20
+    paginate_by = 150
```

Comment:
UX team, please review. This seems to be a bit out of 'design' since this page is much more complicated compared to the others? (same for the subreports)